pkg/pillar/scripts/device-steps.sh

```diff
@@ -191,6 +173,36 @@ else
     echo "$(date -Ins -u) No separate $PERSISTDIR partition"
 fi
 
+DIRS="$CONFIGDIR $CONFIGDIR/DevicePortConfig"
+DIRS="$DIRS $TMPDIR $TMPDIR/DeviceNetworkConfig/ $TMPDIR/AssignableAdapters"
+DIRS="$DIRS $PERSISTDIR $PERSIST_RKT_DATA_DIR $PERSIST_RKT_CONF_LOCAL_DIR $PERSIST_RKT_CONF_LOCAL_AUTH_DIR"
+
+echo "DIRS: $DIRS"
+
+for d in $DIRS; do
+    echo "creating Dir $d"
+    d1=$(dirname "$d")
+    if [ ! -d "$d1" ]; then
+        echo "$(date -Ins -u) Create $d1"
+        mkdir -p "$d1"
+        chmod 700 "$d1"
+    fi
+    if [ ! -d "$d" ]; then
+        echo "$(date -Ins -u) Create $d"
+        mkdir -p "$d"
+        chmod 700 "$d"
+    fi
+done
+
+echo "ls $PERSIST_RKT_CONF_LOCAL_DIR"
```

Comment:
Do we still need this debug output? If so please prepend with the date as for the other echo commands.

Reply:
This can be removed,